docs/general/attributes.md

```diff
@@ -390,6 +390,7 @@ about the span.
 | `code.function` | string | The method or function name, or equivalent (usually rightmost part of the code unit's name). | `serveRequest` | Recommended |
 | `code.lineno` | int | The line number in `code.filepath` best representing the operation. It SHOULD point within the code unit named in `code.function`. | `42` | Recommended |
 | `code.namespace` | string | The "namespace" within which `code.function` is defined. Usually the qualified class or module name, such that `code.namespace` + some separator + `code.function` form a unique identifier for the code unit. | `com.example.MyHttpService` | Recommended |
+| `code.stacktrace` | string | A stacktrace as a string in the natural representation for the language runtime. The representation is to be determined and documented by each language SIG. | `at com.example.GenerateTrace.methodB(GenerateTrace.java:13)\n at com.example.GenerateTrace.methodA(GenerateTrace.java:9)\n at com.example.GenerateTrace.main(GenerateTrace.java:5)` | Recommended |
```

Comment:
Following https://github.com/open-telemetry/opentelemetry-specification/blob/v1.26.0/specification/common/attribute-requirement-level.md I suggest to have the stacktrace as "Opt-In". The commit message already states, that there are only "certain use-cases" that would benefit. Stacktraces usually increase the event size quite substantially and come with a performance cost. Exceptions should occur in a low frequency and greatly benefit from the additional information. Normal events usually do not. Therefore, I feel "Opt-In" would be a more suitable approach.

Reply:
Absolutely agree!

We would move all of the `code` attributes into [the registry](https://github.com/open-telemetry/semantic-conventions/tree/main/model/registry) anyways. And in the context of concrete usages of that field (e.g. in the general attributes semantic conventions we could overwrite it with req-level `Opt-In`)

See this PR: https://github.com/open-telemetry/semantic-conventions/pull/434